src/util.cpp

```diff
@@ -482,6 +482,7 @@ CAboutDlg::CAboutDlg ( QWidget* parent ) : CBaseDlg ( parent )
         "<p>bspeer (<a href=""https://github.com/bspeer"">bspeer</a>)</p>"
         "<p>Martin Passing (<a href=""https://github.com/passing"">passing</a>)</p>"
         "<p>DonC (<a href=""https://github.com/dcorson-ticino-com"">dcorson</a>)</p>"
+        "<p>David Kastrup (<a href=\"https://github.com/dakhubgit\">dakhubgit</a>)</p>"
```

Comment:
@dakhubgit Thanks! 

Would you suggest to use the backslash for the other contributors too?

Reply:
> Would you suggest to use the backslash for the other contributors too?

Good spot! Yes, they should all be changed from `""` to `\"`, as the former is not correct. It is syntactically valid, but is just a concatenation of adjacent strings, with no `"` in the final string:
```
tony@pi:~/jamulus $ strings Jamulus | fgrep Volker
<p>Volker Fischer (<a href=https://github.com/corrados>corrados</a>)</p><p>Peter L. Jones (<a href=https://github.com/pljones>pljones</a>)</p><p>Jonathan Baker-Bates (<a href=https://github.com/gilgongo>gilgongo</a>)</p><p>ann0see (<a href=https://github.com/ann0see>ann0see</a>)</p><p>Daniele Masato (<a href=https://github.com/doloopuntil>doloopuntil</a>)</p><p>Martin Schilde (<a href=https://github.com/geheimerEichkater>geheimerEichkater</a>)</p><p>Simon Tomlinson (<a href=https://github.com/sthenos>sthenos</a>)</p><p>Marc jr. Landolt (<a href=https://github.com/braindef>braindef</a>)</p><p>Olivier Humbert (<a href=https://github.com/trebmuh>trebmuh</a>)</p><p>Tarmo Johannes (<a href=https://github.com/tarmoj>tarmoj</a>)</p><p>mirabilos (<a href=https://github.com/mirabilos>mirabilos</a>)</p><p>Hector Martin (<a href=https://github.com/marcan>marcan</a>)</p><p>newlaurent62 (<a href=https://github.com/newlaurent62>newlaurent62</a>)</p><p>AronVietti (<a href=https://github.com/AronVietti>AronVietti</a>)</p><p>Emlyn Bolton (<a href=https://github.com/emlynmac>emlynmac</a>)</p><p>Jos van den Oever (<a href=https://github.com/vandenoever>vandenoever</a>)</p><p>Tormod Volden (<a href=https://github.com/tormodvolden>tormodvolden</a>)</p><p>Alberstein8 (<a href=https://github.com/Alberstein8>Alberstein8</a>)</p><p>Gauthier Fleutot
</b></p><p>Volker Fischer (<a href=https://github.com/corrados>corrados</a>)</p><p><b>
1Copyright (C) 2005-2020 Volker Fischer and others
```
Some languages allow the doubling of a quote within a string to include a quote, but C++ doesn't.

Need to audit this throughout the code. @ann0see would you like me to do that and raise a PR?

Reply:
Of course, you can open a PR on this.

Reply:
I'd consider this the appropriate way of writing this, yes.